docs/kb/integration/defect-dojo-integration.md

```diff
@@ -0,0 +1,116 @@
+---
+tags:
+  - DefectDojo
+  - Integration
+description: How to connect Semgrep and DefectDojo
+---
+
+# How to connect Semgrep and DefectDojo
+
+[DefectDojo](https://www.defectdojo.com/) is a well-known vulnerability management tool. It allows you to gather security issues from tools like Semgrep and more. By integrating Semgrep findings into DefectDojo, security teams gain additional insight into the security posture of their systems.
+
+## Starting with integration
+Integrate Semgrep with DefectDojo means dumping findings detected by Semgrep into DefectDojo.
+There are several steps to follow:
+
+* In DefectDojo:
+    * Create your product (a product is DefectDojo's concept for a project).
```

Comment:
```suggestion
    1 Create your **product**. A [product](https://defectdojo-dev.readthedocs.io/en/latest/features.html#products) is DefectDojo's term for a project.
```

Just checking - do you mean "project" similar to [GitHub projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects)? That was my first thought. I saw DefectDojo docs about Product, it looks similar but for security management rather than project management.